app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,56 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: %i(edit update)
+  before_action :valid_user, only: %i(edit update)
+  before_action :check_expiration, only: %i(edit update)
+
+  def new; end
+
+  def create
+    @user= User.find_by email: params[:password_reset][:email].downcase
+    if @user
+      @user.create_reset_digest
+      @user.send_password_reset_email
+      flash[:info] = t ".psswd_reset_email"
+      redirect_to root_url
+    else
+      flash.now[:danger] = t ".email_not_found"
+    end
+  end
+
+  def update
+    if params[:user][:password].empty?
+      @user.errors.add :password, t(".nil_psswd")
+      render :edit
+    elsif @user.update_attributes user_params
+      flash[:success] = t ".reset_success"
+      redirect_to login_path
+    else
+      render :edit
+    end
+  end
+
+  def edit; end
+
+  private
+
+  def user_params
+    params.require(:user).permit :password, :password_confirmation
+  end
+
+  def get_user
```

Comment:
Bad naming, you should name a method in ruby using get/set keyword, you can name it like as load_user